iFixit/react-commerce
# Troubleshooting: Use `lite-youtube-embed` to render videos

This is really simple, because the videos are custom elements, so as long as the JS ends up on the page somewhere it should do the trick. We don't have to do any of the finding and triggering on the elements; the browser does that.

## QA Notes
This won't work until https://github.com/iFixit/ifixit/pull/46740 goes out. Once that's done, you should see YouTube videos on /Vulcan/troubleshooting-test.

Closes https://github.com/ifixit/ifixit/issues/46728
